bindings/rust/aws-lc-fips-sys-template/build/main.rs

```diff
@@ -178,17 +182,69 @@ fn prepare_cmake_build(build_prefix: Option<&str>) -> cmake::Config {
 fn build_aws_lc() -> PathBuf {
     let mut cmake_cfg = prepare_cmake_build(Some(&prefix_string()));
 
-    cmake_cfg.build_target("crypto").build()
+    // cmake supports passing multiple arguments to target, but this is broken in the cmake crate
+    // ssl requires crypto so we can get away with just picking the top-most requried one.
+    let target = if cfg!(feature = "ssl") {
+        Some("ssl")
+    } else {
+        Some("crypto")
+    };
+
+    cmake_cfg.build_target(target.unwrap()).build()
```

Comment:
I don't see a need for `Some(...)` here since we unconditionally unwrap it anyways:
```
    let target = if cfg!(feature = "ssl") {
        "ssl"
    } else {
        "crypto"
    };

    cmake_cfg.build_target(target).build()
```